src/vim.h

```diff
@@ -85,9 +85,15 @@
 #endif
 
 /*
- * MACOS_X	    compiling for MacOS X
- * MACOS_X_DARWIN   compiling with the darwin feature for MacOS X
- * MACOS	    compiling for either one
+ * MACOS	    compiling for any of Macintosh operating systems
```

Comment:
I wonder what this is for, now that Classic has been ripped out.  There are now no Macintosh operating systems supported by Vim other than "MACOS_X".